pkg/server/router/router_test.go

```diff
@@ -858,7 +858,7 @@ func BenchmarkService(b *testing.B) {
 			LoadBalancer: &dynamic.ServersLoadBalancer{
 				Servers: []dynamic.Server{
 					{
-						URL: "tchouck",
+						URL: "tchouk",
```

Comment:
I'm unfamiliar w/ this term, but it was used a lot in the repository and there didn't seem to be any indication why it should be written differently here.